regen-network/regen-web
# feat: APP-364 order summary card

## Description

https://regennetwork.atlassian.net/browse/APP-364




---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



https://deploy-preview-2498--regen-marketplace.netlify.app/project/C01-001/buy
Update the credits amount from the order summary card from different steps in the flow and verify the resulting amount to pay updates accordingly.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
